src/main/java/frc/robot/Constants.java

```diff
@@ -239,20 +239,21 @@ public AimModes previous() {
     public static final double lowHeight =
         5.0 + 7.75 / 12.0; // Height of the low goal in ft from the carpet
     public static final double LLHeight =
-        3.0 + 7.0 / 12.0; // Height of the limelight in ft from the carpet
+        28.0 / 12.0; // Height of the limelight in ft from the carpet
     public static final double LLAngle =
-        40.0; // Angle that the limelight is mounted at from a vertical plane, ensure this is as
+        54.0; // Angle that the limelight is mounted at from a vertical plane, ensure this is as
     // exact as possible
 
-    public static final double kP = 6e-4;
-    public static final double kI = 6e-7;
-    public static final double kD = 0.0;
+    public static final double kP = 2.5e-4;
+    public static final double kI = 19e-6;
+    public static final double kD = 0.005;
     public static final double kF = 0.0;
     public static final double kIZone = 0.9;
     public static final double kMaxOutput = 0;
     public static final double kMaxI = 0.9;
     public static final double kMaxISlotId = 0;
     public static final double kMinOutput = 1;
+    public static final double kA = 0.075;
```

Comment:
This feels really small.  The current velocity contributes 0.075 to the SmoothRPM.
The previous velocity contributes ( 0.075)*(1-0.075) = .069 , The one before that contributes ( 0.075)*(1-0.075)^2 = 0.064

We can keep calculating the weights of the previous velocities on the current value...and at 10 values ago, that measurement contributes 0.034.  Which is almost half as impactful as the previous velocity.

To compare, if you set alpha=0.2, then 10 values ago, that velocity contributes 0.02 to the current RPM
If you have output, ,we can look at the impact of different alphas, but I suggest trying at least double what you have.

Reply:
All tuning is subject to change, kA will likely turn out to be around 1.5 but that will require testing.

Reply:
0<= kA <= 1.  1.5 won't work

Agree that it is subject to change, suggest that we increase it now before testing, and confirm on your next testing run.

Reply:
Sorry, .15 is what I meant to send

Reply:
.15 sounds good.  10 values after current, the weight is 0.030